### PR TITLE
lint: fix some lint errors

### PR DESCRIPTION
**Pull request by guits**
This commit fixes the lint errors [301,306]:

Signed-off-by: Guillaume Abrioux <gabrioux@redhat.com>

### CODE REVIEW

File: roles/ceph-dashboard/tasks/configure_dashboard.yml [hunk 1]
```diff
@@ -88,6 +90,7 @@
   delay: 5
   register: ac_result
   delegate_to: "{{ groups[mon_group_name][0] }}"
+  changed_when: false
```
Comment (guits) [inline roles/ceph-dashboard/tasks/configure_dashboard.yml]: this task should be reworked in fact.

File: roles/ceph-dashboard/tasks/configure_dashboard.yml [hunk 2]
```diff
@@ -60,11 +60,13 @@
   shell: |
     {{ container_exec_cmd }} ceph config set mgr mgr/dashboard/server_port {{ dashboard_port }} || \
     {{ container_exec_cmd }} ceph config-key set mgr/dashboard/server_port {{ dashboard_port }}
+  changed_when: false
```
Comment (guits) [inline roles/ceph-dashboard/tasks/configure_dashboard.yml]: if I remember correctly, this is because we want to support multiple ceph release here

Comment (dsavineau) [reply roles/ceph-dashboard/tasks/configure_dashboard.yml]: Yes but it's for pre nautilus release. Considering that the dashboard feature is only stable-4.0+ (so nautilus+) we should only have one command.
BTW this was already mentioned in the inital PR https://github.com/ceph/ceph-ansible/pull/3426#discussion_r280804697

**Comment by guits** [comment]
jenkins test dev-centos-container-all_daemons

**Comment by guits** [comment]
jenkins test dev-centos-container-all_daemons

**Pull request merged**